src/DevTo.Api/DevTo.Api.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.2;netcoreapp3.0</TargetFrameworks>
```

Comment:
Was there a particular reason for changing from .NET Standard 2 to specifically reference .NET Core 2.2 and .NET Core 3.0?

Reply:
re. 1: That's actually a surprise to me, I normally have a version of .NET Framework listed too like I do in my other OSS projects. That said, I'm not entirely against removing supporting .NET Framework support by changing the target.

re. 2: That's pretty weird about the dependencies not working, Newtonsoft.Json in particular directly supports .NET Standard 2.0, I'm guessing it might have been a bug in VS for Mac.

I think for now, reverting this particular change might be best. I might open up another issue about reconsidering targeting and testing particular versions of .NET (eg. whether we should target LTS versions only etc)

Reply:
@Turnerj done!